database/migrations/2022_08_18_143802_create_vendor_employees_table.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('vendor_employees', function (Blueprint $table) {
+            $table->id()->autoIncrement();
+            $table->string('name');
+            $table->string('email')->unique();
+            $table->timestamp('email_verified_at')->nullable();
+            $table->string('activeStatus')->default(true);
```

Comment:
@Babita3331 This seems weird to directly activating a user without sending any kind of verification